### DISCUSSION TITLE
How to incorporate scikit-multilearn with Tensorflow?

### DISCUSSION THREAD

**Discussion by MAXHHHHH** [scikit-multilearn/scikit-multilearn#284] [Q&A] [answered]
Hi,
    I'm using Tensorflow framework to build my multi-label classification model and I want to use some of the loss functions for multi-label classification. I'v tried binary relevance method in Tensorflow with `tf.nn.sigmoid_cross_entropy_with_logits()` and `tf.nn.weighted_cross_entropy_with_logits()` to help build loss function. These two method do works but the results were not so satisfying. So I want to try some other loss functions to help train my model and scikit-multilearn is a nice toolkit.
    So I wonder if is there any API or methods in scikit-multilearn that I can use to train my model in TF
Thank you!

**Comment by niedakh** [comment, marked as answer]
This is not a technical issue. Would you mind asking this question on stackoverflow with a tag "#scikit-multilearn" and providing some definition of what it means that results are not satisfactory? I will gladly answer the question there, please post a link in the reply.
